tests/test_project.py

```diff
@@ -1,6 +1,11 @@
+import os
```

Comment:
Some comments in the testing file would be helpful

Reply:
We will add some comments in this testing file